package.json

```diff
@@ -48,7 +48,7 @@
     "@typescript-eslint/eslint-plugin": "^4.22.0",
     "@typescript-eslint/parser": "^4.18.0",
     "autoprefixer": "^10.2.6",
-    "cheerio": "^1.0.0-rc.6",
```

Comment:
is this still needed? Assuming this can be moved into astro unless other packages use it

Reply:
Tests do use it, but all those tests currently live in `packages/astro` so it could be moved to devDeps there, I suppose. We’ll probably use it there for quite some time, so it’s basically the same effect.